Lib/test/test_pyrepl/test_utils.py

```diff
@@ -60,3 +60,23 @@ def gen_raise():
         self.assertEqual(next(pnw), (3, 4, None))
         with self.assertRaises(ZeroDivisionError):
             next(pnw)
+
+    def test_gen_colors_keyword_highlighting(self):
+        cases = [
+            # no highlights
+            ("a.set", [(".", "op")]),
```

Comment:
Is it possible to add tests with line continuations?

Reply:
will try